host_modules/config_engine.py

```diff
@@ -0,0 +1,55 @@
+"""Config command handler"""
+
+from host_modules import host_service
+import subprocess
+import os
+import shutil
+
+MOD_NAME = 'config'
+DEFAULT_CONFIG = '/etc/sonic/config_db.json'
+
+class Config(host_service.HostModule):
+    """
+    DBus endpoint that executes the config command
+    """
+    @host_service.method(host_service.bus_name(MOD_NAME), in_signature='s', out_signature='is')
+    def reload(self, config_file):
+
+        cmd = ['/usr/local/bin/config', 'reload', '-y']
+        if config_file and config_file != DEFAULT_CONFIG:
+            if not os.path.exists(config_file):
+                return -1, "Can't find %s"%config_file
+            # Persistent Config
+            shutil.move(config_file, DEFAULT_CONFIG)
```

Comment:
Let's move the responsibility to caller side. It is dangerous to destroy default config file, before you reload. The reload could fail.